puppet/modules/quickstack/manifests/pacemaker/nosql.pp

```diff
@@ -92,7 +92,7 @@
     Exec['all-nosql-nodes-are-up'] ->
 
     quickstack::pacemaker::resource::service {'mongod':
-      options        => 'start timeout=10s',
+      options        => 'op start timeout=10s',
       monitor_params => { 'start-delay' => '10s' },
```

Comment:
Also that scenario also utilizes a 120s timeout on the start op rather than the 10s we have here, so we might need to evaluate if we need to increase that value as well.